package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mit-app-inventor/blockly-plugin-workspace-multiselect",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
Maybe let's not bump the version number until we are ready to cut the next release. I think we should try to get in both this PR and #4 for version 0.1.1.

Reply:
Okay, I'm removing this, you can add that version bump and draft the new release when you are ready.